views/tables/ParticipantsTable/index.jsx

```diff
@@ -63,25 +63,46 @@ const ParticipantsTable = (props) => {
   ]
   const cellRenderer = (participant, property) => {
     switch (property) {
+      case 'days':
+        if (participant?.synced && participant?.Consent) {
```

Comment:
If we implement sort we might as well push this logic to the DB using $dateDiff.

Reply:
I've done this implementation, what is the mongodb version we are running on staging..?

[0af550e](https://github.com/AMP-SCZ/dpdash/pull/600/commits/0af550ebd947fb7c731cba9479f9382fa5ea9138)

Reply:
Mongo 5!